src/browser/styles/base.scss

```diff
@@ -1,14 +1,25 @@
 @import 'variables';
 @import 'mixins';
 
+// TODO : pull in some css reset D=
```

Comment:
Always start off with some sort of reset. It'll make your life easier and will prevent a lot of 1-off "margin:0"s in your code.